src/Resource/utils.ts

```diff
@@ -336,3 +341,155 @@ export async function deprecateSelfResource(
     throw error;
   }
 }
+
+export async function getOutgoingLinks(
+  orgLabel: string,
+  projectLabel: string,
+  selfUrl: string,
+  paginationSettings: PaginationSettings,
+): Promise<PaginatedList<ResourceLink>> {
+  return await getLinks(
+    orgLabel,
+    projectLabel,
+    paginationSettings,
+    () => `
+  prefix nxv: <https://bluebrain.github.io/nexus/vocabulary/>
+  SELECT ?total ?o ?p ?self
+
+  WITH {
+    SELECT DISTINCT ?p ?o ?self
+        WHERE {
+            graph ?g1 {
+              <${selfUrl}> ?p ?o
+              FILTER(isIri(?o) && ?p NOT IN (nxv:updatedBy, nxv:createdBy, nxv:constrainedBy, nxv:project, rdf:type))
+            } .
+            OPTIONAL {
+              graph ?g2 {
+              	?o nxv:constrainedBy ?aSchema .
+                ?o nxv:self ?self
+              }
+            }
+      }
+} AS %resultSet
```

Comment:
alignment issue

Reply:
I fought with this for a while and gave up 🤷‍♀️